src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
-import Heading from "./components/Header/Heading";
+import { Route, Routes, Navigate } from "react-router-dom";
 import GalleryList from "./components/Banner/GalleryList";
+import Heading from "./components/Header/Heading";
+import Videos from "./components/Videos";
+import Home from "./components/Home";
+import Leaderboard from "./components/Leaderboard";
+import Challenges from "./components/Challenges";
 import "./App.css";
 
 function App() {
   return (
     <div className="App">
       <Heading />
       <GalleryList />
+      <Routes>
+        <Route path="/" element={<Navigate to="/Home" />} />
+        <Route path="/home" element={<Home />} />
```

Comment:
`<Route path="/" element={<Home />} />`

For now, there is no need for a specific route for home.

Reply:
<NavLink
                to={`/${item.toLowerCase()}`}
                className={({ isActive }) => (isActive ? "active" : "")}
                onClick={() => handleTabChange(item)}
              >
                {item}
              </NavLink> path is given like this. so home is necassary